README.md

```diff
@@ -220,9 +222,13 @@ Note that the public leaderboard uses the test splits for all datasets except MS
 Models should implement the following interface, implementing an `encode` function taking as inputs a list of sentences, and returning a list of embeddings (embeddings can be `np.array`, `torch.tensor`, etc.). For inspiration, you can look at the [mteb/mtebscripts repo](https://github.com/embeddings-benchmark/mtebscripts) used for running diverse models via SLURM scripts for the paper.
 
 ```python
+import mteb
 from mteb.encoder_interface import PromptType
+from mteb.models.wrapper import Wrapper
+import numpy as np
+
 
-class CustomModel:
+class CustomModel(Wrapper):
```

Comment:
Why does it need to inherit from wrapper? I would instead inherit from the Encoder protocol

Reply:
MTEB class will wrap models that not inherit wrapper class https://github.com/embeddings-benchmark/mteb/blob/3ff38ec043f6ebc4e434c083cf96dc823439f8a3/mteb/evaluation/MTEB.py#L366-L367

Reply:
Hmm, maybe it should just wrap SentenceTransformers?

Reply:
It can be like this. You changed it in mieb https://github.com/embeddings-benchmark/mteb/blob/fab0b82a5c2680ab4e4823641955cdc7b9bb9891/mteb/evaluation/MTEB.py#L356-L357 I can change there too

Reply:
Hmm then it will also be merged into v2.0.0 in which case we should probably just update the readme there (I plan to merge it during December)

Reply:
So, for now I think this PR can be merged, and I will update readme in the 2.0 branch